gulpfile.js

```diff
@@ -54,5 +55,22 @@ gulp.task('fonts', function() {
         .pipe(gulp.dest(path.join(output_directory, 'fonts')));
 });
 
+// Extract angular-gettext strings
+gulp.task('pot', function () {
+    return gulp.src(['openslides/core/static/templates/*.html',
+                     'openslides/users/static/templates/users/*.html',
+                     'openslides/users/static/js/users/*.js'])
+        .pipe(gettext.extract('template.pot', {}))
```

Comment:
May be we should rename it so that one can see that the source language is English. Proposal: `en-template.pot` or `template-en.pot`.